Add url_for_static_asset to app and templates (#590)
starlite-api/starlite: docs/usage/15-templating.md

matches [`route_reverse`][starlite.app.Starlite.route_reverse] behaviour. More details about route handler indexing
can be found [here](2-route-handlers/4-route-handler-indexing.md)

> URLs for static files can be created with the `asset_url_path` function. Signature and behaviour is identical to

Reviewer: ```suggestion
URLs for static files can be created using the `asset_url_path` function. It's signature and behaviour are identical to
```

Also consider the name - better to be clear about the use, so I would go with `create_url_for_static_asset`
